docs/getting_started/what_do_i_need.md

```diff
@@ -5,16 +5,16 @@ In order to run Zigbee2MQTT we need the following hardware:
 
 | Name | Price | Picture |
 | ------------- | ------------- | ------------- |
-| Supported Zigbee adapter | 5$ - 40$ | ![CC2531 USB sniffer](../images/cc2531.jpg) |
-| [Supported Zigbee device(s)](../information/supported_devices.md) | - | ![Zigbee devices](../images/xiaomi_sensors.jpg) |
-| PC or Raspberry Pi (*) to run bridge | - | ![Raspberry Pi](../images/pi.jpg) |
+| A supported [Zigbee adapter](../information/supported_adapters.md) (with Zigbee coordinator firmware) | 5$ - 40$ | ![CC2531 USB sniffer](../images/cc2531.jpg) |
+| One or more [supported Zigbee device(s)](../information/supported_devices.md) | - | ![Zigbee devices](../images/xiaomi_sensors.jpg) |
+| A PC or a Raspberry Pi (to run the Zigbee2MQTT software) | - | ![Raspberry Pi](../images/pi.jpg) |
```

Comment:
I think: "A computer (e.g. Raspberry Pi) to run the Zigbee2MQTT software on" is better

Reply:
OK